src/msg/xio/XioConnection.cc

```diff
@@ -77,6 +77,14 @@ void print_ceph_msg(CephContext *cct, const char *tag, Message *m)
   }
 }
 
+#undef dout_prefix
+#define dout_prefix _conn_prefix(_dout)
+ostream& XioConnection::_conn_prefix(std::ostream *_dout) {
```

Comment:
does this method need to have a leading _ in the name?  I'd like to avoid that, if possible.